metricbeat/docs/modules/system.asciidoc

```diff
@@ -26,35 +26,70 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: system
-  period: 10s
   metricsets:
-    - cpu
-    - load
-    - memory
-    - network
-    - process
-    - process_summary
-    #- core
-    #- diskio
-    #- socket
+    - cpu             # CPU usage
+    - filesystem      # File system usage for each mountpoint
+    - fsstat          # File system summary metrics
+    - load            # CPU load averages
+    - memory          # Memory usage
+    - network         # Network IO
+    - process         # Per process metrics
+    - process_summary # Process summary
+    - uptime          # System Uptime
+    #- core           # Per CPU core usage
+    #- diskio         # Disk IO
+    #- socket         # Sockets and connection info (linux only)
+  enabled: true
+  period: 10s
   processes: ['.*']
-  process.include_top_n:
-    by_cpu: 5      # include top 5 processes by CPU
-    by_memory: 5   # include top 5 processes by memory
 
-- module: system
-  period: 1m
-  metricsets:
-    - filesystem
-    - fsstat
-  processors:
-  - drop_event.when.regexp:
-      system.filesystem.mount_point: '^/(sys|cgroup|proc|dev|etc|host|lib)($|/)'
+  # Configure the metric types that are included by these metricsets.
+  cpu.metrics:  ["percentages"]  # The other available options are normalized_percentages and ticks.
+  core.metrics: ["percentages"]  # The other available option is ticks.
 
-- module: system
-  period: 15m
-  metricsets:
-    - uptime
+  # A list of filesystem types to ignore. The filesystem metricset will not
```

Comment:
We decided awhile back ago to strip out most comments from config examples because it meant updating duplicate content and making the docs longer. I think short comments are probably OK, but the comments here seem like that might get out-of-date over time. I worry that we'll forget to update the comments here.

Reply:
This is from our reference docs and are the ones that should be most up-to-date. This part is auto generated.